server/server.js

```diff
@@ -110,15 +117,15 @@ class DataSyncServer {
   async cleanup () {
     await this.models.sequelize.close()
     if (this.schemaListener) await this.schemaListener.stop()
-    if (this.dataSources) await this.disconnectDataSources(this.dataSources)
+    if (this.dataSources) await this.core.disconnectActiveDataSources(this.dataSources)
     if (this.server) await this.server.close()
   }
 
   async onSchemaChangedNotification () {
     log.info('Received schema change notification. Rebuilding it')
     let newSchema
     try {
-      newSchema = await buildSchema(this.models, this.pubsub, this.serverConfig.schemaDirectives)
+      newSchema = await this.core.buildSchema('default', this.pubsub, this.serverConfig.schemaDirectives)
```

Comment:
Just a minor thing: Can we declare this `'default'` value as a constant at the top of the file?

Reply:
This reminds me that we can actually get schema name from pubsub :)